### PR TITLE
YD-327 Move onlyUnreadMessage query parameter to the correct URL

### PR DESCRIPTION
**Pull request by Bert-R**

### CODE REVIEW

File: appservice/src/main/resources/static/swagger/swagger-spec.yaml [hunk 1]
```diff
@@ -531,13 +531,6 @@ paths:
     get:
       summary: Get messages
       description: Fetches all messages related to this activity
-      parameters:
```
Comment (aristotelos) [inline appservice/src/main/resources/static/swagger/swagger-spec.yaml]: Line should be retained (I'll fix this this evening)

File: appservice/src/main/resources/static/swagger/swagger-spec.yaml [hunk 2]
```diff
@@ -1175,6 +1168,13 @@ paths:
       summary: Get messages
       description: Fetches all messages of the user with the given ID
       parameters:
+      parameters:
```
Comment (aristotelos) [inline appservice/src/main/resources/static/swagger/swagger-spec.yaml]: Duplicate `parameters:` tag

**Comment by aristotelos** [comment]
This parameter is missing on week details messages as well

**Comment by Bert-R** [comment]
I moved the ``parameters`` tag back.

The parameter ``onlyUnreadMessages`` only applies to the user messages collection, not to activity messages. I accidentally put the parameter on the wrong URL (day details).

**Pull request merged**